src/components-examples/material/select/select-reactive-form/select-reactive-form-example.html

```diff
@@ -0,0 +1,46 @@
+<form [formGroup]="form">
+  <h4>mat-select</h4>
+  <mat-form-field>
+    <mat-label>First Course</mat-label>
+    <mat-select [formControl]="firstCourseControl" name="food">
+      <mat-option>None</mat-option>
+      <mat-option *ngFor="let food of foods" [value]="food.value">
+        {{food.viewValue}}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
+  <p> First Course: {{firstCourseControl.value}} </p>
```

Comment:
Some extra space before/after the text. It's the same in some of the other `p` tags in this file.